### PR TITLE
APPTS 13.2.0 release note and sign-in/-out verbiage refs

### PR DESCRIPTION
**Pull request by pkclark**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

[Appts 13.2.0 release note and sign-in/-out verbiage refs](https://github.com/Axway/platform-management-docs/pull/136)

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | cf55593 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-137.platform-management-docs.opendocs-builder.pcloud.axway.int |

**Comment by pkclark** [comment]
👌 Cool. The jenkins pass is failing for feature branches off upstream now too. CC @wlai-axway

**Comment by eric34** [comment]
Only thing here I could see is maybe reconsider this: "...users which can authenticate."

**Comment by pkclark** [comment]
Changed to "...users which can sign in with..."

File: content/en/docs/management_guide/managing_accounts/managing_credentials.md [hunk 1]
```diff
@@ -33,19 +33,19 @@ To enable multi-factor authentication:
...
+3. For the **Preferred MFA Method**, select the default method a multi-factor authentication token will be generated for when signing in from a new device. Options are **Authenticator App** and **Email**.
 
     | Preferred MFA method | How this works | When can this be used |
     | ---| --- | --- |
-    | Authenticator App | A random code is generated within the app that needs to be provided when the user logs into the Platform. | Only available when an authenticator app has been configured for your account. See [Setup Authenticator App](#setup-authenticator-app) for details. |
-    | Email | A code is sent to your email address that needs to be provided when the user logs into the Platform. | Always available, uses the email address linked to your account. |
+    | Authenticator App | A random code is generated within the app that needs to be provided when the user signs into the Platform. | Only available when an authenticator app has been configured for your account. See [Setup Authenticator App](#setup-authenticator-app) for details. |
```
Comment (eric34) [inline content/en/docs/management_guide/managing_accounts/managing_credentials.md:40]: "signs in to"

File: content/en/docs/management_guide/managing_accounts/managing_credentials.md [hunk 2]
```diff
@@ -33,19 +33,19 @@ To enable multi-factor authentication:
...
+    | Email | A code is sent to your email address that needs to be provided when the user signs into the Platform. | Always available, uses the email address linked to your account. |
```
Comment (eric34) [inline content/en/docs/management_guide/managing_accounts/managing_credentials.md:41]: "signs in to"

File: content/en/docs/management_guide/managing_accounts/managing_credentials.md (hunk 1 above)
Comment (pkclark) [reply content/en/docs/management_guide/managing_accounts/managing_credentials.md]: Fixed.

File: content/en/docs/management_guide/managing_accounts/managing_credentials.md (hunk 2 above)
Comment (pkclark) [reply content/en/docs/management_guide/managing_accounts/managing_credentials.md]: Fixed.

**Review by eric34 — APPROVED** [review]
Looks good.

**Pull request merged**